### PR TITLE
[ALLUXIO-2327] update javadoc with active voice

### PR DESCRIPTION
**Pull request by nogod1995**
https://alluxio.atlassian.net/browse/ALLUXIO-2327
学号：MG1633061

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/AlluxioShell.java [hunk 1]
```diff
@@ -130,9 +130,8 @@ private void printUsage() {
   }
 
   /**
-   * Method which determines how to handle the user's request, will display usage help to the user
-   * if command format is incorrect.
    *
+   * Handles the specified shell command request, displaying usage if the command format is invalid.
```
Comment (apc999) [inline shell/src/main/java/alluxio/cli/AlluxioShell.java]: could you actually switch line 133 and 134? Otherwise, it looks good to me

**Comment by apc999** [comment]
Jenkins, add to whitelist

**Comment by apc999** [comment]
alluxio-bot, check this please

**Comment by apc999** [comment]
ok to test

**Comment by apc999** [comment]
Also, could you remove `"@nogod1995"` from the title?

**Comment by nogod1995** [comment]
I have changed the PR title and remove the blank notes in shell/src/main/java/alluxio/cli/AlluxioShell.java, thanks for your advice

File: shell/src/main/java/alluxio/cli/AlluxioShell.java [hunk 2]
```diff
@@ -130,9 +130,7 @@ private void printUsage() {
   }
 
   /**
-   * Method which determines how to handle the user's request, will display usage help to the user
-   * if command format is incorrect.
-   *
+   * Handles the specified shell command request, displaying usage if the command format is invalid.
```
Comment (aaudiber) [inline shell/src/main/java/alluxio/cli/AlluxioShell.java]: Can we keep the blank line after the first sentence?

Comment (nogod1995) [inline shell/src/main/java/alluxio/cli/AlluxioShell.java]: Sure, thanks for your advice!

**Comment by aaudiber** [comment]
@nogod1995 Are you planning on reopening this? It looks like the latest build failure was caused by trailing whitespace

**Comment by nogod1995** [comment]
@aaudiber Thanks for your attention. I have try to fix this problem but things become worse，so I start from the beginning,and create a new PR. Thanks again!

**Pull request closed without merging**